Makefile

```diff
@@ -38,7 +38,7 @@ testapps-with-numpy/%: virtualenv
 	$(eval $@_APP_ARCH := $(shell basename $*))
 	. $(ACTIVATE) && cd testapps/on_device_unit_tests/ && \
     python setup.py apk --sdk-dir $(ANDROID_SDK_HOME) --ndk-dir $(ANDROID_NDK_HOME) \
-    --requirements libffi,sdl2,pyjnius,kivy,python3,openssl,requests,sqlite3,setuptools,numpy \
+    --requirements libffi,sdl2,pyjnius,kivy,python3,openssl,requests,sqlite3,setuptools,numpy,opencv,opencv_extras \
```

Comment:
Do we really want to have opencv and extras build each time? Or is it just for temporary testing the PR?

Reply:
hahaha....this was only to test the build...I will reverse it later (that is why I labeled this PR `wip` and put the emoji :construction: in my last commit).

Here are the artifacts for each arch:

https://github.com/kivy/python-for-android/actions/runs/110275077

Btw, it almost doubles the size of the apk...anyway, it may be useful to someone and it doesn't look like hard to maintain.

Reply:
Great that what's I thought, but wasn't sure :+1: 
I have nothing against merging, ping me for the final review